include/common_table.h

```diff
@@ -39,7 +39,7 @@ typedef enum
 { KEY, SERVER } TableType;
 
 typedef enum
-{ OK, ERR_REALLOC, ERR_BUF, ERR_X509, RET_FAIL, NO_MATCH, MEM_ALLOC_FAIL } AddResponseStatus;
+{ OK, ERR, ERR_MEM, ERR_REALLOC, ERR_BUF, ERR_X509, RET_FAIL, NO_MATCH, MEM_ALLOC_FAIL } TableResponseStatus;
```

Comment:
I think some short documentation explaining what each of these means would be helpful.

Reply:
Added some documentation for this enum.